lib/engine/corporation_token.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require_relative 'token'
+
+module Engine
+  class CorporationToken < Token
```

Comment:
keep corporation as token, we should mix 'markers' and tokens. 

let's have another unrelated class called a marker. 

for neutral tokens, if they act as a normal token, they should just be a token with a neutral corporation.

Reply:
they don't block routes but leave them permanently open. I was going to do something like blocks_routes? in the token itself, but I can probably implement that as a check against the 'neutral' corporation.

Neutral corp the best way to go?

Reply:
when a neutral corp is indtantiated it should be passed a no block property which creates non blocking tokens . just a thought

Reply:
Yep, will mod as suggested :)

Reply:
Just as a general discussion. My next step is to actually be able to lay the tokens, I was wanting to use the tile_selector code as a token_selector where the player has a choice of tokens.

This approach okay? It might be useful for other games as well

Reply:
that code could use a refactor, especially the placement and look of the choose and cancel,

it might be appropriate to use that for laying tokens? i'm not sure let's discuss on slack later